ci/pipeline.yaml

```diff
@@ -62,87 +56,43 @@ resources:
       #                         ^^ any prefix (denoting a non-release tag)
       #                           ^ end of the tag string
 
-  - name: build-bundle
-    type: s3
-    icon: zip-disk
+  - name: gh-status
+    type: cogito
     source:
-      bucket: instana-agent-operator-codebuild
-      versioned_file: context.zip
-      access_key_id: ((codebuild-key.key_id))
-      secret_access_key: ((codebuild-key.key_secret))
-      region_name: us-west-2
-      skip_download: true
-
-  - name: codebuild-amd64
-    type: codebuild
-    icon: aws
-    source:
-      project: instana-agent-operator-codebuild
-      region: us-west-2
-      access_key_id: ((codebuild-key.key_id))
-      secret_access_key: ((codebuild-key.key_secret))
-  - name: codebuild-arm64
-    type: codebuild
-    icon: aws
-    source:
-      project: instana-agent-operator-codebuild
-      region: us-west-2
-      access_key_id: ((codebuild-key.key_id))
-      secret_access_key: ((codebuild-key.key_secret))
-  - name: codebuild-s390x
-    type: codebuild
-    icon: aws
-    source:
-      project: instana-agent-operator-codebuild
-      region: us-west-2
-      access_key_id: ((codebuild-key.key_id))
-      secret_access_key: ((codebuild-key.key_secret))
-  - name: codebuild-ppc64le
-    type: codebuild
-    icon: aws
-    source:
-      project: instana-agent-operator-codebuild
-      region: us-west-2
-      access_key_id: ((codebuild-key.key_id))
-      secret_access_key: ((codebuild-key.key_secret))
+      owner: instana
+      repo: instana-agent-operator
+      access_token: ((instanacd-github-api-token))
+      context_prefix: concourse
+      github_host: github.com
 
-  - name: operator-image-amd64
+  - name: agent-operator-image-amd64
     type: registry-image
     icon: docker
     source:
-      repository: gcr.io/instana-agent-qa/instana-agent-operator
-      tag: latest-amd64
-      username: _json_key
-      password: ((project-berlin-tests-gcp-instana-qa))
-  - name: operator-image-arm64
+      repository: delivery.instana.io/int-docker-agent-local/instana-agent-operator/dev-build
```

Comment:
I would vote to add a tag here as
```
tag: ((branch))
```
for all internal images. With this approach we would ensure that we can distinguish between different images in the artifactory UI. This is the approach we have in other PR pipelines